lib/vrl/tests/src/docs.rs

```diff
@@ -37,10 +37,17 @@ pub struct Example {
     source: String,
     #[serde(rename = "return")]
     returns: Option<Value>,
-    output: Option<Event>,
+    output: Option<ExampleOutput>,
```

Comment:
The test framework needed to be updated to account for the fact that `remap` transforms can emit multiple events now.

Reply:
Since this is used for deserializing the JSON, I think the enum is needed here? In the `cue export` the `output` field can either be an array or a single element.